CHANGELOG.md

```diff
@@ -3,12 +3,21 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+
+# 10.5.0-rc.1 April 21, 2022
 ## Features ✨ and improvements 🏁
+* Avoid repeated tile loading from network (or repeated tile decompression when the tile is fetched from the cache database) and repeated vector tile data allocation and parsing when loading render tiles referring to the same logical tile. ([#1282](https://github.com/mapbox/mapbox-maps-android/pull/1282))
+* Switch to use shader to calculate the 'line-trim-offset' property update. ([#1282](https://github.com/mapbox/mapbox-maps-android/pull/1282))
 
 ## Bug fixes 🐞
 * Fix NaN latitude native crash rarely happening during `MapboxMap#flyTo`. ([#1271](https://github.com/mapbox/mapbox-maps-android/pull/1271))
 * Limit `MapboxMap#pixelForCoordinate` to the bounds of MapView. ([#1226](https://github.com/mapbox/mapbox-maps-android/pull/1226))
 * Enable two finger pan gesture. ([#1280](https://github.com/mapbox/mapbox-maps-android/pull/1280))
```

Comment:
this PR is not ported to 10.5 release branch yet, we need to discuss within the team whether to include it in 10.5 release.

Reply:
@pengdev Any reasons why we haven't included to 10.5 yet. imo, since its a bug-fix, we can include it in 10.5.